lib/task-data/schemas/install-photon-os.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "rackhd/schemas/v1/task-schema",
+    "id": "rackhd/schemas/v1/tasks/install-photon-os",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Install Photon OS",
+    "description": "The parameters for Photon OS installation",
+    "describeJob": "Job.Os.Install",
+    "definitions": {
+        "PhotonOsSpecificOptions": {
+            "type": "object",
+            "properties": {
+                "installType": {
+                    "description": "Photon OS install type",
+                    "type": "string",
+                    "enum": [ "minimal", "full" ]
+                }
+            },
+            "required": ["rackhdCallbackScript"]
```

Comment:
This should be removed as no `rackhdCallbackScript` is defined in the properties.